plugins/node.d/icinga

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf
+
+ICINGACLI=${ICINGACLI:-$(which icingacli)}
+JQ=${JQ:-$(which jq)}
+
+
+if [ "$1" = "autoconf" ] ; then
+    if [ "$ICINGACLI" != ""  -a "$JQ" != "" ] ; then
+        echo yes
+        exit 0
+    else
+        echo "no (could not found icingacli or jq)"
+        exit 1
+    fi
+fi
+export TERM=vt100
```

Comment:
Is this really necessary? If yes: please add a comment explaining it.
(otherwise we could never get rid of this line)

Reply:
this was caused by some special characters at the text-output of icingacli. This is not needed after the change to json,